waterloop-site/src/components/SustainableTech/AboutEvent/img/calendar.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="49" height="55" viewBox="0 0 49 55" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can you pass this SVG markup, as well as the other ones, through [SVGOMG](https://jakearchibald.github.io/svgomg/) to clean it up a bit and also prettify?
Also, could you move all SVGs to `waterloop-website-w20/waterloop-site/src/static/img/sustainable-tech`?

Reply:
Sure, I'll make sure to put all SVGs I used through the SVGOMG, and re-locate them to the folder you mentioned